veps/sig-network/decouple-nad.md

```diff
@@ -0,0 +1,178 @@
+# VEP 106: Decouple net-attach-def from KubeVirt 
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [ ] (R) Target version is explicitly mentioned and approved
+- [x] (R) Graduation criteria filled
+
+## Overview
+
+**Target Version**: v1.8
+
+VMs that use secondary networks inject custom resource requests into the virt-launcher pod template whenever custom resources are required to exist on the node in order to fulfill the secondary network.
+One such example is SR-IOV free VFs that must be available on a node for the VM to make use of.
+The names of those custom resources are listed as an annotation in the network-attachment-definition (NAD) CR that corresponds to the secondary network requested by the VM.
+
+Currently, virt-controller queries the NAD CR for every secondary network in every VM that is launched.
+Network-attachment-definition is an external project maintained by the k8s network plumbing WG. The mentioned query is the only direct dependency that KubeVirt production code has on this external project.
+This proposal aims to remove that query and code dependency in favor of an external replacement that will map the custom resources to the virt-launcher pod instead.  
+In addition, the virt-controller role binding that currently allows it to access the NAD can be reduced in scope.
+
+
+## Motivation
+
+The replacement has the architectural advantage of decoupling KubeVirt from an external dependency by simplifying code maintenance and supporting future replacement technologies.
+Security will be improved by removing access permissions to the external CRD.
+In addition, a performance improvement can be gained from using a project that caches NAD data. 
+    
+
+## Goals
+
+NAD query code will be deleted from virt-controller and a replacement injector will be introduced. 
```

Comment:
Please consider the following alternative:
```suggestion
KubeVirt will no longer be aware of NetworkAttachmentDefinition objects.
```

The current wording describes the "how" instead of the "what".